scripts/issues-from-release-notes.py

```diff
@@ -0,0 +1,65 @@
+import argparse
+import json
+import os
+import requests
+
+"""
+Create docs issues from a release notes file.
+
+Before running, add <!--doc--> to the end of each bullet in the release notes file
+that needs a docs issue.
+
+Example use: python3 issues-from-release-notes.py --release_notes=v2.0.1.md --milestone=2.0.x
+
+Requirements:
+  - Requests: http://docs.python-requests.org/en/master/
+  - A GitHub personal access token. This is required for POSTing due to 2-factor auth.
+    Store your token as a GITHUB_ACCESS_TOKEN environment variable.
+"""
+
+parser = argparse.ArgumentParser(
+    description = "create docs issues from a release notes file")
+parser.add_argument("-r", "--release_notes", required=True,
+    help="release notes file from which to create docs issues")
+parser.add_argument("-m", "--milestone", required=True,
+    help="milestone to assign to docs issues, e.g., 2.1, 2.0.x, 2.0")
+
+args = parser.parse_args()
+
+release_notes = args.release_notes
+milestone = args.milestone
+
+# Map milestones to their internal IDs.
+if milestone == "2.1":
+    milestone = 8
+if milestone == "2.O.x":
+    milestone = 10
```

Comment:
One thing to consider is asking GitHub for milestone title -> id mapping: https://developer.github.com/v3/issues/milestones/

Reply:
This is a great idea, but I'll think I'll take it up in a follow-up PR.